src/amo/components/GuidesAddonCard/index.js

```diff
@@ -68,18 +68,20 @@ export class GuidesAddonCardBase extends React.Component<InternalProps> {
                   )}
                 </div>
               </div>
-              <p className="GuidesAddonCard-content-description">
-                {this.props.addonCustomText}
-              </p>
+              <div className="GuidesAddonCard-content-button">
```

Comment:
Why is this called `-button` if the first child is the `description`?

Reply:
Good catch. I've updated the `className`.